CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog for v3.x
 
+## v3.9.2 (2022-12-20)
+
+### Enhancements
+
+  * [migrator] Raise if target version in `to`/`exclusive_to` is not an integer
+  * [mysql] Add support for cross lateral joins
+  * [postgres] Add support for cross lateral joins
+  * [postgres] Add support for materialized CTEs
```

Comment:
@josevalim 

Not sure if you already released, but if you did I think we have the same issue here with materialized CTE. I can make a v3.9 branch to separate it?

Reply:
yeah, i branched from the previous commit :)